src/charm.py

```diff
@@ -153,6 +183,22 @@ def s3_connection_info(self) -> Optional[S3ConnectionInfo]:
             bucket=raw_info.get("bucket"),
         )
 
+    @property
+    def db_connection_info(self) -> Optional[DatabaseConnectionInfo]:
```

Comment:
Following  @welpaolo comment, it is probably best to rename the property to refer to `metastore`

Reply:
Updated to `metastore_db_connection_info`.